certified-connectors/TALXISDataFeed/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1216 @@
+{
+  "definitions": {
+    "DataFeed.Model.Entities.Holidays": {
+      "type": "object",
+      "properties": {
+        "global": {
+          "type": "boolean",
+          "description": "Is this public holiday in every county (federal state)"
+        },
+        "fixed": {
+          "type": "boolean",
+          "description": "Is this public holiday every year on the same date"
+        },
+        "types": {
+          "description": "Type of public holiday",
+          "enum": [
+            "Public",
+            "Bank",
+            "School",
+            "Authorities",
+            "Optional",
+            "Observance"
+          ],
+          "type": "string"
+        },
+        "launchYear": {
+          "description": "The launch year of the public holiday",
+          "format": "int32",
+          "type": "integer"
+        },
+        "date": {
+          "description": "Date of the holiday",
+          "format": "date",
+          "type": "string"
+        },
+        "localName": {
+          "type": "string",
+          "description": "Local name of the holiday"
+        },
+        "name": {
+          "type": "string",
+          "description": "International name of the holiday"
+        },
+        "counties": {
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "countryCode": {
+          "type": "string",
+          "description": "Country code"
+        }
+      }
+    },
+    "GoogleApi.Entities.Places.Common.StructuredFormatting": {
+      "type": "object",
+      "properties": {
+        "mainText": {
+          "type": "string"
+        },
+        "mainTextMatchedSubstrings": {
+          "items": {
+            "$ref": "#/definitions/GoogleApi.Entities.Places.Common.MatchedSubstring"
+          },
+          "type": "array"
+        },
+        "secondaryText": {
+          "type": "string"
+        },
+        "secondaryTextMatchedSubstrings": {
+          "items": {
+            "$ref": "#/definitions/GoogleApi.Entities.Places.Common.MatchedSubstring"
+          },
+          "type": "array"
+        }
+      }
+    },
+    "DataFeed.Model.Entities.Company.CompanyFinance": {
+      "type": "object",
+      "properties": {
+        "revenue": {
+          "type": "object",
+          "properties": {
+            "average": {
+              "title": "Gross Revenue (Average)",
+              "format": "double",
+              "type": "number"
+            },
+            "upperBound": {
+              "title": "Gross Revenue (Upper Bound)",
+              "format": "double",
+              "type": "number"
+            },
+            "lowerBound": {
+              "title": "Gross Revenue (Lower Bound)",
+              "format": "double",
+              "type": "number"
+            },
+            "year": {
+              "title": "Gross Revenue Year",
+              "description": "",
```

Comment:
@honzakostejn Empty property values can be removed until unless you have a specific reason to keep them.

Reply:
I'll delete them.